backend/README.md

```diff
@@ -30,6 +30,14 @@ In `keys.json`:
 }
 ```
 
+However, `keys.json` is no longer used as part of the login. We use the following accounts as test logins:
+
+  * member@test.com
+  * lead@test.com
+  * director@test.com
```

Comment:
I think it's safer to not include `director@test.com` since directors have rights have access to our auth server and change it so if someone runs this and uses that login they could potentially make changes to our orgs users.